packages/core/utils/lib/sanitize/utils.js

```diff
@@ -0,0 +1,15 @@
+'use strict';
+
+const pipeAsync = (...methods) => async data => {
+  let res = data;
+
+  for (const method of methods) {
+    res = await method(res);
+  }
+
+  return res;
+};
```

Comment:
I think this could be a reduce function. But it's mostly cosmetic.

Reply:
I considered it but in this scenario, I find using plain old loops simpler to read than smth like that

```javascript
const result = await methods.reduce(async (acc, method) => {
  return await method(acc);
  …
}, data);
```
Thanks for the input though :slightly_smiling_face: